backend/routers/controllers/message_controllers/topic-service.js

```diff
@@ -44,6 +44,26 @@ function findUserTopics(userLogin) {
     },
   });
 }
+function findTopicsOnPage(page) {
+  const limit = 10;
+  const offset = (page-1) * limit;
```

Comment:
You'll find first 30 topics if you're looking for page three, and not topics from 20 through 30, is that intended?

Reply:
offset - starting point
limit - number of loops
if you loking for 3 page,there will be starting point (offset) 2*10=20 and number of topics 10
its like ```for (i=offset;i<=offset+limit;i++)```